### PR TITLE
Fixes for artillery, WP and some FSM level tweaks

### PR DESCRIPTION
**Pull request by nk3nny**
- Fix for artillery https://github.com/nk3nny/LambsDanger/issues/34 #36
- Fix to WP descriptions https://github.com/nk3nny/LambsDanger/issues/9
- Tweaks to CQC speed
- Tweaks to Suppression time (based on feedback)
- Changed 'Disable AI' variable to 'lambs_danger_disableAI'
- Minor addition to fnc_immediateActionEXP  pending discussion.
- Adds buttons for: https://github.com/nk3nny/LambsDanger/issues/35
- Reassess Radio ranges, adds CBA settings: https://github.com/nk3nny/LambsDanger/issues/25
- Removed remoteExec on advice from diwako (I will look into this) 
- Added three keys: (This could be moved to its own PBO)
	Toggle AI
	Quick Suppression
	Quick Hide 
- Added more use of internal findOverwatch function

### CODE REVIEW

**Comment by nk3nny** [comment]
Just testing another minor feature: 
- AI to disembark static weapons when player is close

File: addons/danger/functions/fnc_immediateActionExp.sqf [hunk 1]
```diff
@@ -13,23 +13,27 @@
...
 params ["_unit","_pos"];
 
-// Standing or recent explosions ignored
-if !(stance _unit isEqualTo "STAND" || {_unit getVariable [QGVAR(isExploded),0] > time}) exitWith {false};
+// Standing or recent explosions ignored // probably want to add more checks for captive states or similar static behaviour.
+if (
+    isplayer _unit 
+    || {stance _unit isEqualTo "PRONE"}
+    || {_unit getVariable [QGVAR(isExploded),0] > time}
+) exitWith {false};
```
Comment (nk3nny) [inline addons/danger/functions/fnc_immediateActionExp.sqf]: This file is WIP. 
Discussion at https://github.com/nk3nny/LambsDanger/issues/37

Comment (diwako) [reply addons/danger/functions/fnc_immediateActionExp.sqf]: As discussed, this should be most likely be removed or off loaded into its own addon module.

File: addons/danger/CfgEventHandlers.hpp [hunk 2]
```diff
@@ -5,6 +5,7 @@ class Extended_PreStart_EventHandlers {
 };
 class Extended_PreInit_EventHandlers {
     class ADDON {
+        clientInit = QUOTE(call COMPILE_FILE(XEH_preInitClient));
```
[the remark this answers is not in the document]
Comment (nk3nny) [reply addons/danger/CfgEventHandlers.hpp]: @jokoho48 
Name change noted. It should still be initialised under 'Extended_PreInit_EventHandlers'?

[the remark this answers is not in the document]
Comment (diwako) [reply addons/danger/CfgEventHandlers.hpp]: if you want it to run at preinit, yes.

[the remark this answers is not in the document]
Comment (jokoho48) [reply addons/danger/CfgEventHandlers.hpp]: sorry i missed that it is in PreInit XEH. so that is fine

[the remark this answers is not in the document]
Comment (nk3nny) [reply addons/danger/CfgEventHandlers.hpp]: It defines some buttons.  If there is no functional benefit from running it as a preInit, then either is fine by me. *shrug*

**Pull request merged**